From 0deecaa8eb7ccbec5a383bf370778d303e828b82 Mon Sep 17 00:00:00 2001
From: Steve deRosier <Steve.Derosier@lairdtech.com>
Date: Thu, 22 May 2014 04:54:08 -0700
Subject: [PATCH] laird_fips: remove dead header to fix x86 compilation

<asm/system.h> is a deprecated header that has been removed from x86
kernels already. moddebug.c doesn't actually use it, so kill it. This fixes
a compilation failure on x86.

Bug: 5592
---
 drivers/net/wireless/laird_fips/moddebug.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/wireless/laird_fips/moddebug.c b/drivers/net/wireless/laird_fips/moddebug.c
index bf61ead..9957aef 100644
--- a/drivers/net/wireless/laird_fips/moddebug.c
+++ b/drivers/net/wireless/laird_fips/moddebug.c
@@ -19,7 +19,6 @@
 #include <linux/sched.h>
 #include <linux/spinlock.h>
 
-#include <asm/system.h>		/* cli(), *_flags */
 #include <asm/uaccess.h>	/* copy_*_user */
 
 #include "moddebug.h"
-- 
1.8.3.2

